### ISSUE TITLE
Title and role attributes are missing for functional control on Dashboard screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#797]
**Describe the bug**
Title and role attributes are missing for functional control on Dashboard screen.  When we move mouse over different Dashboard Gadgets, there is no label/title displayed in some cases.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash&pathType=unknown)
2. Hover over "Add Dashboard Gadgets". 
3. No title/label displayed.
4. Same thing happens if we add different Dashboard Gadgets on the Dashboard. For eg. If we add a dashboard Gadget such as "Comment" and hover over "Moderation Default" then no title is displayed. 

**Expected behavior**
Title/Label should be displayed on each component of the screen.

**Comment by rgarg24** [comment]
This issue has been fixed while working on different bug.
